ledger/ledger-api-integration-tests/src/test/itsuite/scala/com/digitalasset/platform/tests/integration/ledger/api/TransactionServiceIT.scala

```diff
@@ -725,7 +756,7 @@ class TransactionServiceIT
 
       "accept exercising a well-authorized multi-actor choice with coinciding controllers" in allFixtures {
         c =>
-          val List(operator, receiver, giver) = List(party1, party2, party3)
+          val List(operator, _, giver) = List(party1, party2, party3)
```

Comment:
```suggestion
          val List(operator, receiver @ _, giver) = List(party1, party2, party3)
```

Reply:
I'll gladly fix this but can you explain why?